test/lib/data_hygiene/bulk_organisation_updater_test.rb

```diff
@@ -7,7 +7,9 @@ def process(csv_file)
     file.close
 
     begin
-      DataHygiene::BulkOrganisationUpdater.call(file.path, logger: Rails.logger)
+      with_current(user: create(:admin_user)) do
+        DataHygiene::BulkOrganisationUpdater.call(file.path, logger: Rails.logger)
```

Comment:
Good catch. Although the original implementation seems a bit weird. I wonder if it would be clearer to have two methods on `EventLog`: one for recording an `EMAIL_CHANGED` event and one for recording an `EMAIL_CHANGE_INITIATED` event...? What do you think...?

Separately, I've been experimenting with setting `Current.user` to a sensible value for rake tasks, use of the rails console, and for background jobs. I wonder if this might be a better way to solve this problem (because `Current.user` would then always be set.